tests/test.py

```diff
@@ -56,9 +56,14 @@ def __init__(self, args: list[str]):
         self.log_dir = tempfile.TemporaryDirectory()
         self.output_file = os.path.join(self.log_dir.name, "fatrace.log")
         self.log_content: str | None = None
+        self.stderr_content: str | None = None
 
         fatrace_bin = "fatrace" if os.getenv("FATRACE_INSTALLED_TEST") else str(ROOTDIR / "fatrace")
-        self.process = subprocess.Popen([fatrace_bin, "-o", str(self.output_file)] + args)
+        self.process = subprocess.Popen(
+            [fatrace_bin, "-o", str(self.output_file)] + args,
+            stdout = subprocess.PIPE,
+            stderr = subprocess.PIPE,
```

Comment:
Not a fan, this makes test failures harder to debug. But I also think that only catches the "too many dirs" warning, which really should go away.

Reply:
Reverted in non-recursive rewrite.